docker/safety-rules/Dockerfile

```diff
@@ -17,7 +17,7 @@ FROM toolchain AS builder
 COPY . /libra
 
 ENV RUSTFLAGS "-Ctarget-cpu=skylake -Ctarget-feature=+aes,+sse2,+sse4.1,+ssse3"
-RUN cargo build --release -p libra-node -p cli -p config-builder -p safety-rules && cd target/release && rm -r build deps incremental
+RUN cargo build --release -p libra-node -p cli -p config-builder -p libra-key-manager -p safety-rules && cd target/release && rm -r build deps incremental
```

Comment:
Copy this exact build command to the other Docker files (this is done for docker caching, so that the cargo build is only done once across all the images).

    docker/client/Dockerfile:RUN cargo build --release -p libra-node -p cli -p config-builder -p safety-rules && cd target/release && rm -r build deps incremental
    docker/init/Dockerfile:RUN cargo build --release -p libra-node -p cli -p config-builder -p safety-rules && cd target/release && rm -r build deps incremental
    docker/mint/Dockerfile:RUN cargo build --release -p libra-node -p cli -p config-builder -p safety-rules && cd target/release && rm -r build deps incremental
    docker/validator/Dockerfile:RUN cargo build --release -p libra-node -p cli -p config-builder -p safety-rules && cd target/release && rm -r build deps incremental

Reply:
Ooh -- that's a cool trick. Each build is pretty long, so it's cool to know we have that 😄